src/lib/localStorageKeys.ts

```diff
@@ -1,5 +1,5 @@
 export const preferencesKey = 'note.boostio.co:preferences'
-export const storageDataListKey = 'note.boostio.co:storageDataList'
+export const storageDataListKey = 'note.boostio.co:localStorageDataList'
```

Comment:
@Rokt33r  anything to be careful about with this change? Users need to load all storages again. We should probably Unsupported storage page was already discarded, so maybe some message or similar to tell user that the loaded storage is not supported?